server/contracts.go

```diff
@@ -0,0 +1,70 @@
+package server
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+
+	emulator "github.com/onflow/flow-emulator"
+	"github.com/onflow/flow-go-sdk"
+	"github.com/onflow/flow-go-sdk/templates"
+	"github.com/onflow/flow-go/fvm"
+	"github.com/onflow/flow-nft/lib/go/contracts"
+	fusd "github.com/onflow/fusd/lib/go/contracts"
+)
+
+var baseContractsPath = "./server/contracts/"
+
+type DeployDescription struct {
+	name        string
+	address     flow.Address
+	description string
+}
+
+func deployContracts(conf *Config, b *emulator.Blockchain) []DeployDescription {
+	ftAddress := flow.HexToAddress(fvm.FungibleTokenAddress(b.GetChain()).Hex())
+	serviceAcct := b.ServiceKey().Address
+
+	contracts := map[string][]byte{
+		"FUSD":             fusd.FUSD(ftAddress.String()),
+		"NonFungibleToken": contracts.NonFungibleToken(),
+		"ExampleNFT":       contracts.ExampleNFT(serviceAcct.Hex()),
+		"NFTStoreFront": loadContract("NFTStorefront.cdc", map[string]flow.Address{
+			"FungibleToken":    serviceAcct,
+			"NonFungibleToken": serviceAcct,
+		}),
+	}
+	for name, contract := range contracts {
+		templates.AddAccountContract(serviceAcct, templates.Contract{
+			Name:   name,
+			Source: string(contract),
+		})
+	}
+
+	addresses := []DeployDescription{
+		{"FUSD", serviceAcct, "💵  FUSD contract"},
+		{"NonFungibleToken", serviceAcct, "✨   NFT contract"},
+		{"ExampleNFT", serviceAcct, "✨   NFT contract"},
+		{"NFTStorefront", serviceAcct, "✨   NFT contract"},
+	}
+
```

Comment:
Use emulator instance `SendTransaction` function here.

Reply:
oh!!! will give that a try -- right now i'm just using the raw blockchain methods :P

![image](https://user-images.githubusercontent.com/30219253/139911796-bcde9160-f360-43e4-a041-d890309ef53d.png)

Reply:
You can just use `CreateAccount()` method on blockchain and avoid handling this yourself.

Reply:
Yeah you are right sorry, no method like that on the blockchain.

Reply:
Ah, okay! I think this'll do it (now it actually checks the service account before printing, to make sure!)

![image](https://user-images.githubusercontent.com/30219253/140002555-153a09fa-d835-4f69-a69a-8ec7f3fd3b5c.png)